rotkehlchen/chain/ethereum/decoding/constants.py

```diff
@@ -38,3 +38,6 @@
 
     string_to_ethereum_address('0x340d693ED55d7bA167D184ea76Ea2Fd092a35BDc'): 'Uphold.com',
 }
+
+GAS_COUNTERPARTY = 'gas'
```

Comment:
If you check the code you will see, we use `CPT` as prefix for all counterparties.

```suggestion
CPT_GAS = 'gas'
```